backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -265,27 +264,40 @@ public TestOrder editQueueItem(
 
   private TestResult getCovidResultFromMultiplexList(List<DiseaseResult> results) {
     return results.stream()
-        .filter(result -> result.getDiseaseName().equals("COVID-19"))
+        .filter(r -> r.getDiseaseName().equals("COVID-19"))
         .findFirst()
-        .orElseThrow(() -> new IllegalArgumentException("No COVID-19 test result found"))
-        .getTestResult();
+        .map(DiseaseResult::getTestResult)
+        .orElse(null);
   }
 
   @AuthorizationConfiguration.RequirePermissionUpdateTestForTestOrder
   public TestOrder editQueueItemMultiplex(
       UUID testOrderId, UUID deviceSpecimenTypeId, List<DiseaseResult> results, Date dateTested) {
-    TestOrder savedOrder =
-        editQueueItem(
-            testOrderId,
-            deviceSpecimenTypeId,
-            getCovidResultFromMultiplexList(results).toString(),
-            dateTested);
+    lockOrder(testOrderId);
```

Comment:
Just a note that I edited these functions to be a pretty direct copy of the original `editQueueItem` and `addTestResult`. Nick had originally referenced those functions and just used these new functions to update the non-covid test results, but with the corrections and removals logic added it became much easier to simply duplicate the entire function and update the Results in a single place. After these have been in place for a while we can remove the old methods.